src/test/java/core/basesyntax/service/RecordParserTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import core.basesyntax.database.Database;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class RecordParserTest {
+    private static final int EXPECTED_OK_SIZE = 2;
+    private static RecordParser recordParser;
+
+    @BeforeClass
+    public static void beforeAll() {
+        recordParser = new RecordParserImpl();
+    }
+
+    @Test
+    public void parseRecords_correctData_Ok() {
+        List<String> testData = new ArrayList<>();
+        testData.add("b,banana,20");
+        testData.add("b,apple,100");
+        recordParser.parseRecords(testData);
+        int actual = Database.RECORDS.size();
+        int expected = EXPECTED_OK_SIZE;
```

Comment:
In my opinion, your parser should return parsed data. But here you compare some data with the result of the **_recordDao.addRecord()_** method.

Reply:
I made this method according to SRP video on MA platform. I compare this data because parser returns nothing and calls Dao addition method.